HDP.md

```diff
@@ -129,6 +129,12 @@ Returns information on a single game.
     Rounds => uint
     InitialFood => uchar
     MaxSerpents => uchar
+    Walls => NumWalls [Cell]
+    NumWalls => uint
```

Comment:
I would've used an unsigned short, since the rows and cols are uchars, the maximum amount of walls would be 255^2 (65536) while the max ushort is 65535. So unless we need to send a map where every tile is a wall, then a short is enough by definition.